docs/manual/java/gettingstarted/java-prereqs.md

```diff
@@ -0,0 +1,75 @@
+# Lagom Java prerequisites
+
+Before starting with Lagom, check to make sure that you have the correct versions of a Java Development Kit (JDK) and build tool. Your build tool must also have internet access. 
+
+Reference these sections for help with:
+
+1. [JDK](#jdk)
+1. [Maven](#maven) or [sbt](#sbt)
+1. [Internet access through a proxy](#internet-access-through-a-proxy)
+
+
+
+## JDK
+
+Before installing a build tool, verify that you have a Java Development Kit (JDK), version 8 or higher and that your machine is configured correctly.
+
+Check the JDK version by running `java` and `javac` from the command line:
+
+```
+java -version
+javac -version
+
+```
+For `java`, the console should respond with the major version number of 1.8, the following shows an example of what you would see in the shell with a 1.8.0_74-b02 build:
+```
+java version "1.8.0_74"
+Java(TM) SE Runtime Environment (build 1.8.0_74-b02)
```

Comment:
Ignasi, I decided to leave off the command prompt to make it universal regardless of which shell the reader is using. For example, I used a Windows command shell when I tried these commands the first time. That is why I separated the commands from the output.